sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/AmqpExceptionUtil.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.implementation;
+
+import com.azure.core.amqp.exception.AmqpException;
+import com.azure.core.amqp.exception.ErrorCondition;
+
+import java.time.Duration;
+
+class AmqpExceptionUtil {
+    /**
+     * Base sleep wait time.
+     */
+    private static final int SERVER_BUSY_BASE_SLEEP_TIME_IN_SECS = 4;
+
+    /**
+     * Check if the existing exception is a retriable exception.
+     *
+     * @param exception An exception that was observed for the operation to be retried.
+     * @return true if the exception is a retriable exception, otherwise false.
+     */
+    static boolean isRetriableException(Exception exception) {
```

Comment:
Doesn't this move potentially make it more difficult for custom retry policies to receive guidance on whether an exception is considered for retries?